### PR TITLE
added manifest

### PR DESCRIPTION
**Pull request by bazylevnik0**
### Please confirm your submission meets all the criteria

- [+] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [+] My pull request follows the instructions at [App Submission][submission].
- [+] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [+] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [+] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [+] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [+] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "id" : "org.gnome.Convolution",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "44",
+  "runtime-commit" : "bcaf01ac12cddc49dd34d1c186f0d19cab9fb01ded47e063eb39327ef4602945",
```
Comment (hfiguiere) [inline manifest.json:5]:
```suggestion
```

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "sdk" : "org.gnome.Sdk",
+  "sdk-commit" : "ea8595e019f7442effb1d73f42a3231d46913ffb5aa5ae50a7ede53fe94619ea",
```
Comment (hfiguiere) [inline manifest.json:7]:
```suggestion
```

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "id" : "com.github.bazylevnik0.Convolution",
```
Comment (hfiguiere) [inline manifest.json:2]: as per the documentation it should be `io.github`

File: manifest.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      "sources" : [
+        {
+          "url" : "https://gitlab.gnome.org/bazylevnik0/convolution.git",
+          "commit" : "0a980125543f5b1041446ae5891bc5684e898b91",
+          "type" : "git"
+        }
+      ]
+    }
+  ],
+  "finish-args" : [
```
Comment (hfiguiere) [inline manifest.json:32]: `finish-args` should be above `modules`

File: manifest.json (hunk 3 above)
Comment (hfiguiere) [reply manifest.json:2]: also technically since it is not hosted on github, there might be questions.

Comment (hfiguiere) [reply manifest.json:2]: `org.gnome.gitlab` is the base

one doesn't simply make up IDs. `com.github` will be rejected anyway.

Comment (hfiguiere) [reply manifest.json:2]: gitlab is NOT a username. So the latter.

File: manifest.json
Comment (barthalion) [inline manifest.json:1]: The filename has to match the ID plus extension.

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.bazylevnik0.Convolution

**Comment by hfiguiere** [comment]
The appstream file doesn't have a `summary`

**Comment by hfiguiere** [comment]
it is actually insufficient

**Comment by hfiguiere** [comment]
> I think fixed...

there was no change so there is nothing to review.

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.bazylevnik0.Convolution

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.bazylevnik0.Convolution

**Comment by hfiguiere** [comment]
```
builddir/export/share/metainfo/org.gnome.gitlab.bazylevnik0.Convolution.metainfo.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.bazylevnik0.Convolution

**Pull request closed without merging**